### ISSUE TITLE
Frontend exits on docker-compose up

### ISSUE THREAD

**Issue by WillowP** [ohduran/cookiecutter-react-django#345]
**Describe the bug**
I'm trying to get set up with this awesome cookiecutter on MacOS on an M1 MacBook. The frontend fails to start.

**To Reproduce**
1. Run the cookiecutter
2. Run `$ docker-compose up`

**Expected behavior**
Both containers start.

**Screenshots**
Here's the terminal output from after I run `docker-compose up`
```
Starting project_name_django_1 ... done
Starting project_name_react_1  ... done
Attaching to project_name_django_1, project_name_react_1
django_1  | Watching for file changes with StatReloader
django_1  | Performing system checks...
django_1  | 
django_1  | System check identified no issues (0 silenced).
django_1  | August 20, 2021 - 15:29:12
django_1  | Django version 4.0, using settings 'backend.settings.development'
django_1  | Starting development server at http://0.0.0.0:8000/
django_1  | Quit the server with CONTROL-C.
react_1   | 
react_1   | > frontend@0.1.0 start
react_1   | > react-scripts start
react_1   | 
react_1   | ℹ ｢wds｣: Project is running at http://172.18.0.3/
react_1   | ℹ ｢wds｣: webpack output is served from 
react_1   | ℹ ｢wds｣: Content not from webpack is served from /app/public
react_1   | ℹ ｢wds｣: 404s will fallback to /
react_1   | Starting the development server...
react_1   | 
project_name_react_1 exited with code 0
```

**Desktop (please complete the following information):**
 - OS: MacOS
 - Version 11.5.2
 - Apple M1

**Additional context**
`npm i` `npm start` works just fine, but can't connect to the backend because the settings seem to be expecting docker.

**Comment by cristianmedeiros** [comment]
Followed this thread and it solved: https://github.com/facebook/create-react-app/issues/8688

TL;DR;
add `stdin_open: true` to docker-compose and it worked

**Comment by ohduran** [comment]
Thank you @cristianmedeiros for taking the time to look at this issue. Will associate to the PR shortly.
